AndelaOSP/art-backend
# #159460356 Add field to track whether asset details are verified.

#### What does this PR do?
 - Add a field called `verified` on the Asset model.
 - Update the admin site to allow for filtering listed asset on their verified status.

#### How should this be manually tested?
- Navigate to the admin site, asset section;  a column called "verified" should show.
- Create a new asset; a checked checkbox labeled "verified" should shown on the form.

#### Relevant pivotal tracker stories?
[#159460356](https://www.pivotaltracker.com/story/show/159460356)

#### Screenshots
<img width="1652" alt="screen shot 2018-08-02 at 16 09 56" src="https://user-images.githubusercontent.com/10899666/43585823-b5dcbffa-966e-11e8-9fc7-855eb3964977.png">

> Admin site showing ability to filter based on the verification status of an asset.
